lib/MusicBrainz/Server/Data/Role/OptionsTree.pm

```diff
@@ -32,6 +32,38 @@ sub get_tree {
     return $root;
 }
 
+=method is_child
+
+Checks whether a specific row id is a child of the given one in the tree. 
+
+=cut
+
+sub is_child {
+    my ($self, $own_id, $possible_child_id) = @_;
+
+    # _table for link attribute types includes a bunch of joins we don't want
+    my $table = $self->_type eq 'link_attribute_type'
+        ? 'link_attribute_type'
+        : $self->_table;
```

Comment:
Getting the table from `%ENTITIES` would be a bit more future-proof in case we add other joins in the future.

Reply:
That'd require the table to be on `%ENTITIES` in the first place, which seems fairly rare - but I guess I will just fix that.